package.json

```
@@ -66,7 +66,7 @@
     "async": "^0.9.0",
     "bytebuffer": "^3.5.4",
     "coveralls": "^2.11.2",
-    "dox": "^0.6.1",
+    "dox": "^0.7.0",
```

Old dox was silently failing.